_topic_maps/_topic_map.yml

```diff
@@ -1290,6 +1290,8 @@ Topics:
       File: ovn-k-anp
     - Name: About BaselineAdminNetworkPolicy
       File: ovn-k-banp
+    - Name: Metrics for AdminNetworkPolicy and BaselineAdminNetworkPolicy
```

Comment:
same like with nodes/networks do we really need to everytime mention its ANP and BANP ? its all under "admin network policy" section which makes the whole thing self explanatory

Reply:
Changing to just AdminNetworkPolicy